### ISSUE TITLE
Any way to remove file based on CDN path instead of cloudPath?

### ISSUE THREAD

**Issue by ghost** [rossj/rackit#3]
Hi, I have my application store the CDN path to the database instead of using cloudPath, so every time there's a server request, I can provide the file link right away instead of converting to the cloudPath first.

The issue is, every time I want to update the file through the website, the process is to remove the old file first before adding the new one with new random name, therefore I need the last file path in cloudPath to do that. Since I don't save the cloudPath for database size reason (I only store 1 necessary data, which is the CDN path), I could not get the cloudPath required to remove the old data.

In short, Is there any way I could get the cloudPath using a CDN path instead? So I could remove files based on CDN path, or simply use the CDN path right away. So its the opposite of the getURI function.

Thanks.
